packages/xo-web/src/xo-app/backup/health/index.js

```diff
@@ -67,6 +67,9 @@ const ACTIONS = [
   },
 ]
 
+@addSubscriptions({
+  schedules: subscribeSchedules,
+})
```

Comment:
We should make sure that no snapshots is shown before we have a list of schedules.